layouts/GeneralLayout.tsx

```diff
@@ -33,7 +33,7 @@ const GeneralLayout = ({ children }: { children: ReactNode }) => {
           className={clsx(
             'flex flex-col min-h-full px-4 pb-2 md:ml-[290px] lg:w-[calc(100%-290px)]',
 
-            pathname != '/' && ' transition-color ease-in-out duration-200',
+            pathname != '/' && ' transition-color ease-in-out duration-0',
```

Comment:
If we don't need the transition, please remove these classes.

```suggestion
```

Reply:
Thank you for your suggestion, @aftabrehan. I will get it done.